opm/autodiff/StandardWell.hpp

```diff
@@ -84,7 +87,11 @@ namespace Opm
         // the index for Bhp in primary variables and also the index of well control equation
         // they both will be the last one in their respective system.
         // TODO: we should have indices for the well equations and well primary variables separately
-        static const int Bhp = numWellEq - numWellControlEq;
+        static const int Bhp = numStaticWellEq - numWellControlEq;
+
+        // total number of the welll equations and primary variables
```

Comment:
Typo: welll -> well.

Also while here: "there is no extra well equations will be used" -> "no extra well equations will be used".